docroot/themes/humsci/humsci_basic/src/scss/components/_pager.scss

```diff
@@ -131,6 +131,21 @@
 
         }
 
+        .hs-button,
```

Comment:
Works and look good. Just a small detail to be on the safe side: can you please update a selector to include the case when the `<a>` is inside another element with the button classes? Something like this:

```
.hs-button, .hs-button a,
.hs-button--big, .hs-button--big a,
...
```

I don't think this is being used in any site right now, but it's a valid pattern for buttons and it's better to build the styles as bulletproof as possible.